packages/core/src/controllers/ApiController.ts

```diff
@@ -138,6 +138,16 @@ export const ApiController = {
     }
   },
 
+  getChains() {
+    const { caipNetwork } = NetworkController.state
+    if (CoreHelperUtil.checkCaipNetwork(caipNetwork, 'solana')) {
```

Comment:
this seems to be unnecessary, doesn't it return the same string as `caipNetwork.id`?